test/sol6/kyberNetwork.js

```diff
@@ -1018,10 +1018,92 @@ contract('KyberNetwork', function(accounts) {
                     await expectRevert(
                         network.tradeWithHint(networkProxy, srcToken.address, srcQty, ethAddress, taker,
                             maxDestAmt, minConversionRate, platformWallet, hint),
-                        "0 rate"
+                        "trade invalid, if hint involved, try parseHint API"
                     );
                 });
             };
+
+            describe("should return 0 rate for unlisted reserve", async() => {
```

Comment:
should return 0 rate when trying to trade a token with a reserve that is not listed with this token.

Reply:
change as request